src/main/java/org/icgc/argo/workflow_management/service/api_to_wes/impl/ApiProducerToWes.java

```diff
@@ -47,7 +47,7 @@ public ApiProducerToWes(Sender<WfMgmtRunMsg> sender) {
   @Override
   public Mono<RunsResponse> run(RunsRequest runsRequest) {
     val runId = generateWesRunId();
-    val msg = createWfMgmtRunMsg(runId, runsRequest, RunState.INITIALIZING);
+    val msg = createWfMgmtRunMsg(runId, runsRequest, RunState.QUEUED);
```

Comment:
does this mean we will always go `QUEUED` -> `INITIALIZING` no matter what the setup? If so does everything else play nice with this? I think the answer to both of these are yes but just want to make sure to ask the question.

Reply:
Yea thats what I'm thinking. Workflow-API (once its ready) will just send out QUEUED events to mgmt domain i.e. the rabbitmq exchange. It doesn't need to worry about if gatekeeper is really their or not and then managment will be configured to accept QUEUED or INITIALIZING based on if gatekeeper is on. I guess with out gatekeeper management doesn't QUEUE, it goes straight to INITIALIZING kinda like how CANCELLING for QUEUED goes straight to CANCELLED... I need to update executeConsumer to handle this actually

Reply:
Also I think we might still need to go to canceling first even from Queued ... It's still an async chain with the first step being ack the cancel request and the second actually cancelling